### DISCUSSION TITLE
Meeting Minutes 3/23/23

### DISCUSSION THREAD

**Discussion by DevinCLane** [timmyichen/dev-directory#52] [Meeting Minutes]
## Agenda
- Week in review - what happened? What got shipped? 
  - #41 
  - #47 
  - #38 
  - #51 
  - #44
- What's in-flight - what do people need to get unblocked? 
  - #49 
  - #50 - some merge conflicts from #44
  - #12 - this one is open!
  - #33 
  - #39 - this is unblocked now @jmoldyvan 
- The week ahead - 
  - please add comments on persistence of pagination via the URL in https://github.com/timmyichen/dev-directory/discussions/17
  - please review https://github.com/timmyichen/dev-directory/pull/50
- Next meeting: @wo1vin to run things

## Parking Lot Items

**Friendly reminder from the group** - don't be afraid to ask for help! If you're stuck on something for longer than an hour, stop and ask to pair with someone. We're all friendly here and still learning. It's very likely that when you pair with someone, both of you will be learning something new. And it's fun! So lean into it often!

## Action Items

**Comment by timmyichen** [comment]
Re what's in store for the week ahead - I'd love to get the next two tech specs (Pagination via @DevinCLane and Onboarding via @jmoldyvan) turned into tickets!  The profiles project is probably wrapping up soon so we'll need some of the scoped work turned into individual pieces of work for people to pick up :)

If either of y'all want me to help out with the issue-writing, happy to do so - just let me know! I know your schedules are tight and this is but a volunteer project haha

**Reply by DevinCLane** [reply]
Yes! I would love to pair on turning these into tickets, I have time tomorrow then I'll be out until 4/3, if you had time @timmyichen

**Reply by DevinCLane** [reply]
also if it doesn't get done before I'm gone, hopefully someone else can pick up turning them into tickets so it's not blocking

**Reply by jmoldyvan** [reply]
Yeah I'd be happy to set up a time and turn them into tickets with some help!

**Reply by timmyichen** [reply]
@DevinCLane when do you have time friday?  earlier is better for me because timezones

@jmoldyvan does that work for you too? otherwise we can set aside time for later

**Reply by DevinCLane** [reply]
@timmyichen will message on Discord

**Comment by Natashka333** [comment]
hello everyone ! I have created an interview script. I would appreciate it if anyone can take a look and give me any feedback. My first tryyyy 

https://docs.google.com/document/d/1UL9hHzv50JyhxvWbjTh5GMZzMDH8VHlu4Ch-Ou8fDqM/edit?usp=sharing

it is open for your suggesting :)
thanks !
